File: matiasmasso/Mat.NET/DT2/DTOStaffCategory.vb
Public Class DTOStaffCategory
    Inherits DTOBaseGuid

    Property segSocialGrup As DTOSegSocialGrup
    Property nom As String
    Property ord As Integer

    Public Sub New()
        MyBase.New
    End Sub

    Public Sub New(oGuid As Guid)
        MyBase.New(oGuid)
    End Sub
End Class

File: matiasmasso/Mat.NET/DTO/DTOStaffCategory.cs
using System;

namespace DTO
{
    public class DTOStaffCategory : DTOBaseGuid
    {
        public DTOSegSocialGrup SegSocialGrup { get; set; }
        public string Nom { get; set; }
        public int Ord { get; set; }

        public DTOStaffCategory() : base()
        {
        }

        public DTOStaffCategory(Guid oGuid) : base(oGuid)
        {
        }
    }
}
